api/server/router/local/local.go

```diff
@@ -0,0 +1,161 @@
+package local
```

Comment:
`local` implementation. I used it because we also use `local` in the volume drivers as a `this is the driver that comes built in`.